tests/acceptance/tests/ProductAnalytics/ValidatePRAEvents.spec.ts

```diff
@@ -0,0 +1,362 @@
+        await AdminOrderDetail.saveButton.click();
+        const request = await requestPromise;
+        expect(request.url()).toContain(PRODUCT_ANALYTICS_ENDPOINT);
+
+        await ShopAdmin.expects(AdminOrderDetail.contextMenuButton).toBeVisible()
+    });
+
+    await test.step('Navigate via page view to dashboard page', async () => {
+
+        const requestPromise = AdminDashboard.page.waitForRequest(`**/${PRODUCT_ANALYTICS_ENDPOINT}`);
+        await ShopAdmin.goesTo(AdminDashboard.url());
+        const request = await requestPromise;
+        expect(request.url()).toContain(PRODUCT_ANALYTICS_ENDPOINT);
+
+        await ShopAdmin.expects(AdminDashboard.adminMenuOrder).toBeVisible();
+        // eslint-disable-next-line playwright/no-conditional-in-test
+        if (!await isSaaSInstance(TestDataService.AdminApiClient)) {
+            await ShopAdmin.expects(AdminDashboard.welcomeHeadline).toBeVisible();
+        }
+    });
+
+    await test.step('Validate captured requests for product analytics', async () => {
+
+        const events = parseCapturedEvents(captured);
+        expect(events).toHaveLength(8);
+
+        const eventIds = events.map(e => e.event_id);
+        expect(eventIds).toEqual([1, 2, 3, 4, 5, 6, 7, 8]);
+
+        const eventTypes = events.map(e => e.event_type);
+        expect(eventTypes).toEqual([
+            'Link Visited',   // event_id 1
+            'Page Viewed',    // event_id 2
+            'Page Viewed',    // event_id 3
+            'Page Viewed',    // event_id 4
+            'Link Visited',   // event_id 5
+            'Page Viewed',    // event_id 6
+            'Button Click',   // event_id 7
+            'Page Viewed',    // event_id 8
+        ]);
+
+        const [
+            firstLinkVisited,    // event_id 1
+            pageViewed,          // event_id 2
+            pageViewed2,         // event_id 3
+            pageViewed3,         // event_id 4
+            linkVisited,         // event_id 5
+            pageViewedDetail,    // event_id 6
+            buttonClicked,       // event_id 7
+            pageViewedBackToDash,// event_id 8
+        ] = events;
+
+        // ----------------------
+        // event_id = 1: first Link Visited (dashboard -> order listing)
+        // ----------------------
+        const firstLinkVisitedProps = firstLinkVisited.event_properties;
+
+        expect(firstLinkVisitedProps.sw_link_href).toBeDefined();
+        expect(firstLinkVisitedProps.sw_link_type).toBeDefined();
+        expect(firstLinkVisitedProps.sw_page_path).toBeDefined();
+        expect(firstLinkVisitedProps.sw_page_name).toBeDefined();
+
+        expect(firstLinkVisitedProps.sw_link_href).toBe('#/sw/order/index');
+        expect(firstLinkVisitedProps.sw_link_type).toBe('internal');
+        expect(firstLinkVisitedProps.sw_page_path).toBe('/sw/dashboard/index');
+        expect(firstLinkVisitedProps.sw_page_name).toBe('sw.dashboard.index');
```

Comment:
IMO the first block of checks is not necessary. If the properties are not set, the second check against the string will fail.